src/posts/lucky_0.17.cr

```diff
@@ -0,0 +1,41 @@
+class Lucky016Release < BasePost
+  title "Lucky 0.17 has been released!"
+  slug "lucky-0_17-release"
+
+  def published_at
+    Time.utc(year: 2019, month: 8, day: 16)
+  end
+
+  def summary
+    <<-TEXT
+    Lucky v0.17 is a huge release with many enhancements to Avram, Lucky's
+    database library for Postgres.
+    TEXT
+  end
+
+  def content
+    <<-MD
+    Lucky v0.17.0 is now out and has support for the newest Crystal (v0.30.0)!
```

Comment:
This is the most pedantic thing I've ever pointed out but I think either this should not have the trailing `.0` or we should include it in the `summary`

```suggestion
    Lucky v0.17 is now out and has support for the newest Crystal (v0.30.0)!
```